include/seqan3/alignment/configuration/align_config_aligned_ends.hpp

```diff
@@ -211,11 +211,35 @@ second_seq_trailing(value_t) -> second_seq_trailing<value_t>;
  * \details
  *
  * A wrapper for providing ordered access to the end-gap specifiers independent of the input order.
- * The possible input types can be: seqan3::first_seq_leading, seqan3::first_seq_trailing, seqan3::second_seq_leading and
- * seqan3::second_seq_trailing.
+ * The possible input types can be: seqan3::first_seq_leading, seqan3::first_seq_trailing, seqan3::second_seq_leading
+ * and seqan3::second_seq_trailing.
  * The types in the parameter pack `ends_t` are deduced by the corresponding constructor argument.
- * If a specifier is not set it will default to `false` and thus the respective end-gap will be penalized in the
+ * If a specifier is not set it will default to `false` and thus the respective end-gap will be penalised in the
  * pairwise alignment.
+ *
+ * ### Static vs runtime configuration
+ *
+ * The end_gaps class preserves the static/non-static property of the respective end-gap specifier. Those specifiers
+ * can, depending on how they are constructed, contain a static information or a runtime information whether or not a
+ * specific end-gap is enabled. To check whether the information was static the function end_gaps::is_static can be
+ * used. If it was static the function end_gaps::get_static can be used to obtain the respective value at compile time.
+ * Note, that both functions are static and thus need to be called on the type and not on an instance of the type.
+ *
+ * \snippet snippet/alignment/configuration/aligned_ends_example.cpp access
+ *
+ * To get the respective value at runtime use `[]`-operator. This function returns always the respective value
+ * independent of whether the value was provided by a static variable or a runtime variable.
+ * Also note that static and non-static end-gap specifier can be mixed within the seqan3::end_gaps class template.
```

Comment:
```suggestion
 * Also note that static and non-static end-gap specifier can be mixed within the end_gaps class template.
```

In this file you can either link with `end_gaps` (because it's in the file) or `seqan3::align_cfg::end_gaps` (the full identifier).